src/main/asciidoc/javadsl/main.adoc

```diff
@@ -130,3 +131,47 @@ $ tail -f stdout_1.log
 2017-11-22 18:04:08.647  INFO 26655 --- [r.woodchuck-1-1] log-sink          : wood
 ----
 Note that the partitioning is done based on the hash of the `java.lang.String` object.
+
+==== Step 5 Use Authentication
+Optionally if you have enabled authentication on a server there are 3
+different ways to authorize a client.
+
+Use basic authentication:
```

Comment:
Could we maybe include the `current` link of SCDF's security docs that list these 3 options?

Reply:
Looks like we don't have any docs as in shell we created and documented those shortcuts like `--dataflow.username`. Oauth option were added just for tile and never documented. Java DSL just points to microsite and we don't have anything there either.